cvmfs/loader.cc

```diff
@@ -55,6 +55,7 @@ struct CvmfsOptions {
   int grab_mountpoint;
   int cvmfs_suid;
   int disable_watchdog;
+  int fast_parse;
```

Comment:
Maybe clarify in the name of that variable that you deal with the option parsing? `fast_option_parsing` or `minimal_option_parsing`?